DirectEmployers/MyJobs
# Lowered loading overlay below main nav

## What's this PR do?
Lowers loading overlay below main nav. Users can now access tabs at any view size.

## Screenshots (if appropriate)
![screen shot 2017-07-05 at 2 57 29 pm](https://user-images.githubusercontent.com/25237191/27880238-49b015d2-6192-11e7-875d-96264ed8c68d.png)

## Where should the reviewer start?
Custom CSS > .loading-overlay { z-index }

## How should this be manually tested?
Pause action in redux dev tools, change view size. The overlay should stay below the main nav.

## Any background context you want to provide?
